manifests/agent.pp

```diff
@@ -76,6 +77,35 @@
     module_helper  => $module_helper,
   }
 
+  package {'psmisc':
```

Comment:
Not sure how you feel about this. I'm using `killall` to trigger all pxp-agent reloading their log file.